src/sentry/utils/services.py

```diff
@@ -101,7 +101,7 @@ def expose(self, context):
         base = self._base
         base_instance = base()
         for key in itertools.chain(base.__all__, ("validate", "setup")):
-            if inspect.ismethod(getattr(base_instance, key)):
+            if inspect.isroutine(getattr(base_instance, key)):
```

Comment:
I just replaced uses of `ismethod` elsewhere. I don't know that these were specifically broken, but likely doesn't hurt.

Reply:
This should be fine, although these should be methods by definition since line 102 above instantiates the type. I don't really see a drawback to it being overly permissive, though.